src/scribe_data/check/check_project_structure.py

```diff
@@ -73,7 +73,32 @@
 BASE_DIR = "../language_data_extraction"
 
 
-def check_data_type_folders(path, language, subdir, errors):
+def check_for_sparql_files(folder_path, data_type, language, subdir, missing_queries):
+    """
+    Check if a data-type folder contains at least one .sparql file.
+
+    Args:
```

Comment:
@axif0 Can you please remember to format your docstrings in a way that's consistent with the rest of the project. As it's written this won't be parsed into the documentation.

Reply:
Sorry for this. And thank you. I will remember this in future.